kate_phillips.cfdg

```diff
@@ -0,0 +1,33 @@
+startshape FISH
+
+shape FISH
```

Comment:
Did the ordering of FISH with its subcomponents matter in terms of first do tail, then do Body, and so on? I had difficulty with layering when I was trying to test things out.